### PR TITLE
Adapt and extend EOS WET basis

### PR DESCRIPTION
**Pull request by dvandyk**
- add sbsb operators
 - rename "b->qlnul" -> "qblnul"
 - add ubtaunutau and cbtaunutau
 - add missing "tex" item

### CODE REVIEW

File: wet.eos.basis.json [hunk 1]
```diff
@@ -2,9 +2,75 @@
...
   },
   "sectors": {
+    "sbsb": {
+      "sbsb::Re{c1}": {
+        "real": true,
+        "tex": "\\frac{4G_F}{\\sqrt{2}} (V_{tb} V_{ts}^*)^2 (\\bar s_L \\gamma^\\mu b_L)(\\bar s_L \\gamma_\\mu b_L)"
+      },
+      "sbsb::Im{c1}": {
+        "real": true,
+        "tex": "\\frac{4G_F}{\\sqrt{2}} (V_{tb} V_{ts}^*)^2 (\\bar s_L \\gamma^\\mu b_L)(\\bar s_L \\gamma_\\mu b_L)"
```
Comment (DavidMStraub) [inline wet.eos.basis.json:15]: If this is a real WC, there is an i (or rather -i) missing in the operator's tex (since it's defined as Lagrangian = -CO).

Comment (dvandyk) [reply wet.eos.basis.json:15]: The wilson coefficient is complex-valued in general. However, in EOS I use only real-valued parameters, which requires expressing this as two separate parameters in wcxf. This is identical to, e.g., the treatment of ``C_9`` in ``b->smumu``.
Neither my approach nor yours with the factor of ``i`` is wrong, I think. I declare one operator, and split the complex-valued WC into two-real valued parameters. You use two linearly dependent definitions of the operator, and keep only the real-part of their Wilson coefficients.

For what it's worth, there is precedent for my way, but I'm happy to convinced to use your way!

File: wet.eos.basis.json [hunk 2]
```diff
@@ -216,171 +282,256 @@
...
+        "real": true,
+        "tex": "-\\frac{4G_F}{\\sqrt{2}} V_{cb} (\\bar c_R b_L)(\\bar \\tau_R \\nu_{\\tau L})"
+      },
+      "cbtaunutau::Im{cSL}": {
+        "real": true,
+        "tex": "-\\frac{4G_F}{\\sqrt{2}} V_{cb} (\\bar c_R b_L)(\\bar \\tau_R \\nu_{\\tau L})"
+      },
+      "cbtaunutau::Re{cT}": {
+        "real": true,
+        "tex": "-\\frac{4G_F}{\\sqrt{2}} V_{cb} (\\bar c_R \\sigma^{\\tau\\nu} b_L)(\\bar \\tau_R \\sigma_{\\tau\\nu}\\nu_{\\tau L})"
```
[the remark this answers is not in the document]
Comment (dvandyk) [reply wet.eos.basis.json:403]: Thanks for catching that. I must have accidentally/mindlessley appended ``/g`` to my regex!

File: wet.eos.basis.json (hunk 1 above)
Comment (DavidMStraub) [reply wet.eos.basis.json:15]: You're right, my suggestion was wrong - I re-read the format definition and `real: true` is only admissible for Hermitian operators, so also using an i does not work. So the only way to conform with the standard is to use a complex WC with `Re` and `Im` keys. I understand that you only use real parameters, but what's the difference between having two separate keys and single key with `Re` and `Im` as subkeys (as required by the standard)?

Comment (dvandyk) [reply wet.eos.basis.json]: I had forgotten about the Re and Im part, and nobody caught this error until now. Have adapted the basis, and I will correct  ``wilson'' once this has been merged.

Comment (DavidMStraub) [reply wet.eos.basis.json]: Yes, it had slipped my attention as well.

**Comment by dvandyk** [comment]
Danke @DavidMStraub!

**Pull request merged**